src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import Page from './Page';
```

Comment:
```suggestion
import React, { useState } from 'react';

import Page from './Page';
```

저는 코드 가독성을 위해 의미상으로 구분되는 경우 빈줄을 넣어줘요.
코드를 보면 빈줄을 넣어주는 경우가 있는데 자신만의 기준이있나요?

사실 컴퓨터의 입장에서는 빈줄이 필요하지 않아요.
하지만 의미상으로 구분해서 빈줄을 넣어주면 코드를 읽는 프로그래머에게는 도움이되죠 ;)

Reply:
저는 가독성을 위해 메소드는 한칸씩 공백을 사용해서 구분을 지었는데 의미상으로 구분하진 않았습니다. 트레이너님 말씀대로 역할에 따라 빈공백을 쓰는 습관을 들여보겠습니다:)